### PR TITLE
Upgrade gcloud dependency to google-cloud 0.19.x

### PR DESCRIPTION
**Pull request by bjwatson**
Fixes https://github.com/googleapis/artman/issues/103 and https://github.com/googleapis/artman/issues/102.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 oslo.utils>=3.4.0 # Apache-2.0
 pyyaml>=3.11
-taskflow==1.25.0
+taskflow>=1.25.0,<2.0.0
```
Comment (bjwatson) [inline requirements.txt]: I'm just fixing this to be consistent with `setup.py`

**Comment by bjwatson** [comment]
Never mind. This is more involved than I thought. I reverted the offending change from master, so the urgency is somewhat reduced.

There's still a version conflict in the self-service instructions that originally motivated what I was doing here.

**Comment by shinfan** [comment]
Travis seems cannot import "google.cloud". Is this module name incorrect?

**Comment by geigerj** [comment]
@shinfan what's especially strange is that the import error only happens in the `pylint` environment, not `pep8` or `py27`...

**Comment by geigerj** [comment]
@shinfan From the Travis log:

```
pylint-errors installed:...
...google-cloud==0.19.0...
```

So `google-cloud` should be installed available in the `pylint-errors` environment. And

```
$ virtualenv venv
$ . venv/bin/activate
(venv) $ pip install google-cloud==0.19.0
  ...
(venv) $ python
Python 2.7.12 (default, Aug 24 2016, 08:19:55) 
[GCC 4.8.4] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import google.cloud
>>>
```

So the module `google.cloud` name seems OK.

**Comment by shinfan** [comment]
Umm this is strange...Can you reproduce the issue locally?

**Comment by lukesneeringer** [comment]
This is _way_ out of date at this point. Declaring it dead.

**Pull request closed without merging**